### PR TITLE
[IMP] inventory: multi-step label section

### PR DESCRIPTION
**Pull request by Felicious**
## Summary
- Add label printing configs for 2 and 3 step delivery
- slight modifications to "Get Rate" window
- Update screenshots for version 17.0

Fwport: yes!

[Task](https://www.odoo.com/web#id=3873658&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by Felicious** [comment]
Hello @ksc-odoo ! This PR is ready for your first round of peer review (: It's a small feature update

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst [hunk 1]
```diff
@@ -26,14 +27,45 @@ labels. Finally, provide the company's :ref:`source address
...
 =====================
 
-Tracking labels are generated after the delivery order (DO) is validated.
+To configure automatic printing of tracking labels, add the third-party carrier to the sales order
+(SO) in the *Sales* app. Then, validate the delivery order (DO) or the relevant operation to trigger
+the label to be automatically printed.
+
+.. note::
+   For companies using two or three step delivery, refer to the :ref:`printing labels for multi-step
+   delivery <inventory/shipping_receiving/picking-config>` section to print the label during the
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:63]:
```suggestion
   delivery <inventory/shipping_receiving/picking-config>` section to learn how to print the label during the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst [hunk 2]
```diff
@@ -54,8 +86,12 @@ click the :guilabel:`Add Shipping` button in the bottom-right corner of the quot
    :alt: Show the "Add Shipping" button on the quotation.
 
 In the resulting pop-up window, select the intended carrier from the :guilabel:`Shipping Method`
-drop-down menu. Clicking :guilabel:`Get Rate` displays the shipping cost for the customer, via the
-third-party carrier in the :guilabel:`Cost` field.
+drop-down menu. The :guilabel:`Total Order Weight` field is automatically populated based on the
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:89]:
```suggestion
drop-down menu. The :guilabel:`Total Order Weight` field is automatically populated, based on the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst [hunk 3]
```diff
@@ -54,8 +86,12 @@ click the :guilabel:`Add Shipping` button in the bottom-right corner of the quot
...
+:ref:`weight of products in the order <inventory/shipping_receiving/configure-weight>`. Modify this
+field to overwrite the predicted weight and use this weight to estimate the cost of shipping.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:91]:
```suggestion
field to overwrite the predicted weight, and use this weight to estimate the cost of shipping.
```

**Review by ksc-odoo — APPROVED** [review]
@Felicious -- just finished my Peer Review on this. Looks good to me! Only a handful of suggestions that require your attention. once you implement the necessary changes, feel free to move it to the next round. Thanks! 👍

**Comment by Felicious** [comment]
Hello @StraubCreative, this ~50 line PR is ready for your final review! :D Tenks for your help

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst [hunk 4]
```diff
@@ -26,14 +27,45 @@ labels. Finally, provide the company's :ref:`source address
...
+      :align: center
+      :alt: Enable the "Print Label" feature.
+
 Print tracking labels
 =====================
 
-Tracking labels are generated after the delivery order (DO) is validated.
+To configure automatic printing of tracking labels, add the third-party carrier to the sales order
+(SO) in the *Sales* app. Then, validate the delivery order (DO), or the relevant operation, to
+trigger the label to be automatically printed.
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:59]: This section seems to be both redundant and in conflict with lines 66-71. 
Consider condensing in a way where all of the conditions are covered, but in one paragraph, with one clear set of instructions for each condition.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst [hunk 5]
```diff
@@ -54,8 +86,12 @@ click the :guilabel:`Add Shipping` button in the bottom-right corner of the quot
    :alt: Show the "Add Shipping" button on the quotation.
 
 In the resulting pop-up window, select the intended carrier from the :guilabel:`Shipping Method`
-drop-down menu. Clicking :guilabel:`Get Rate` displays the shipping cost for the customer, via the
-third-party carrier in the :guilabel:`Cost` field.
+drop-down menu. The :guilabel:`Total Order Weight` field is automatically populated, based on the
+:ref:`weight of products in the order <inventory/shipping_receiving/configure-weight>`. Modify this
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:90]: Good addition here 👍

**Review by StraubCreative — APPROVED** [review]
Hi @Felicious 
Looks good to me! 
I just had one comment for you to consider, otherwise this is ready for @samueljlieber when you're ready 🙂

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst (hunk 4 above)
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:59]: Addressed this comment with the following revision (:

```rst
 Tracking labels are printed when specific operations are validated. By default, validating a
delivery order (DO) generates a tracking label in the chatter.

.. note::
   For companies using two or three step delivery, refer to the :ref:`printing labels for multi-step
   delivery <inventory/shipping_receiving/picking-config>` section to learn how to print the label
   after validating a picking or packing operation.

When both the *Sales* and *Inventory* apps are installed, begin in the :menuselection:`Sales` app,
and proceed to the desired quotation or sales order (SO). There, and :ref:`add the shipping cost
<inventory/shipping_receiving/add-shipping-quote>` to the order. Then, navigate to the linked |DO| —
or another operation type when using multi-step delivery — to validate the operation and print the
label.

If only the *Inventory* app is installed, create :abbr:`DOs (Delivery Orders)` directly in the
:menuselection:`Inventory` app , :ref:`add the third-party carrier
<inventory/shipping_receiving/validate-print-label>` in the :guilabel:`Carrier` field, and validate
the |DO|.
```

**Comment by Felicious** [comment]
Hey @samueljlieber ! This PR is ready for your technical review, when you get the chance (:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst [hunk 6]
```diff
@@ -26,14 +27,46 @@ labels. Finally, provide the company's :ref:`source address
...
+   after validating a picking or packing operation.
+
+When both the *Sales* and *Inventory* apps are installed, begin in the :menuselection:`Sales` app,
+and proceed to the desired quotation or sales order (SO). There, and :ref:`add the shipping cost
+<inventory/shipping_receiving/add-shipping-quote>` to the order. Then, navigate to the linked |DO| —
+or another operation type when using multi-step delivery — to validate the operation and print the
+label.
 
 If only the *Inventory* app is installed, create :abbr:`DOs (Delivery Orders)` directly in the
 :menuselection:`Inventory` app , :ref:`add the third-party carrier
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst:72]: Nitpick– while were at it, can you please take care of this extra space 🙏 😄
```suggestion
:menuselection:`Inventory` app, :ref:`add the third-party carrier
```

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious! Nice work on this PR. Your changes looks great to me, and Im approving and delegating merge to you, with one small fix. Thank you! 😊
..
@robodoo delegate=Felicious

**Comment by Felicious** [comment]
@robodoo r+

**Comment by Felicious** [comment]
@robodoo r+

**Comment by d-fence** [comment]
robodoo retry
qunit failing now skipped

**Pull request closed without merging**